### PR TITLE
Add carousel

### PR DESCRIPTION
**Pull request by vzajkov**
## Description
Add React Slick carousel to allow snap to center for team member profiles.

## Test Plan
Clicking on a team member profile that is not in center should center their profile photo.

### CODE REVIEW

**Comment by karenmrose** [comment]
@vzajkov This is looking good! A couple of things:

- On the desktop version, it looks like the highlighted picture is in the center on Safari, but it's not centered on Chrome 

This is what it looks like on Chrome:
Screen Shot 2019-12-17 at 10 53 23 AM

- For the mobile view, the image seems to be off center. The image should be centered on the device. 
Screen Shot 2019-12-17 at 10 35 54 AM

**Comment by vzajkov** [comment]
@karenmrose Added the fixes requested, as well as another small issue I noticed where scrolling was not showing the relevant bio of the team member. Was able to confirm views are now centered on Firefox and Chrome, as well as mobile.

File: client/js/hah-app/team.js [hunk 1]
```diff
@@ -28,42 +32,78 @@ class Team extends Component {
...
+				// {
+				// 	breakpoint: 1300,
+				// 	settings: {
+				// 		slidesToShow: 5,
+				// 		slidesToScroll: 1,
+				// 		infinite: true,
+				// 	},
+				// },
+				{
+					breakpoint: 1300,
```
Comment (karenmrose) [inline client/js/hah-app/team.js:57]: Let's use the standard breakpoints. I just committed [this change to master](https://github.com/HelpAssistHer/help-assist-her/commit/acdc542ae8f8550b10c3464e5ca344391624555f), so if you merge that in, you can actually just use the variable and don't have to hardcode the breakpoint values.

Comment (karenmrose) [reply client/js/hah-app/team.js:57]: Can you also remove the commented out code if it's not going to be used?

**Comment by karenmrose** [comment]
It looks like the jumpiness has been fixed! The scrolling looks good. 

Just one more design thing: it looks like the width in between the pictures is a little too wide.
Screen Shot 2020-04-26 at 10 57 48 AM

If you look at how it's formatted now, the margin is smaller: https://hah-prod.herokuapp.com/about

**Comment by vzajkov** [comment]
@karenmrose I've removed the comments and reduced the white space between the images - however, I don't think we can use the standard breakpoints here and keep the images close together in all view sizes. At 1100px, for example - we would need to go from showing 3 slides to 5 slides, but 5 slides would not fit. I haven't been able to find a good solution to this (and I've looked at other libraries like pure-react-carousel), but the work around I've found to this is by adjusting the `centerPadding` to keep the closeness of the slides consistent across all views.

**Pull request merged**